torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1236,12 +1225,7 @@ void SpecialPostProcess(Node* n) {
       };
 
       if (seq_node && t_type && t_type->scalarType()) {
-        if (seq_node->kind() == prim::ListConstruct &&
```

Comment:
Where do we check for this? Also curious as to where the recursive calling in happening

Reply:
This case shouldn't happen because for pattern of `prim::ListConstruct -> aten::insert`, it directly converts to `onnx::SequenceEmpty -> onnx::SequenceInsert`.

The recursive calling happens for patterns where `prim::ListConstruct` is passed through blocks (through loop) until it reaches `onnx::SequenceInsert`.